api/handlers_resourcenames.go

```diff
@@ -0,0 +1,134 @@
+package api
```

Comment:
This is still part of the space handlers IMO, I don't think it warrants a new file.

Reply:
:+1: